### PR TITLE
CI: fix syntax error in codeql.yml

### PR DESCRIPTION
**Pull request by neteler**
Attempt to fix syntax error:

```
Check failure on line 46 in .github/workflows/codeql.yml
GitHub Actions / CodeQL

Invalid workflow file

The workflow is not valid. .github/workflows/codeql.yml (Line: 46, Col: 17): Unexpected value ''
```

See log: https://github.com/mundialis/github-workflows/actions/runs/9971374622/workflow

I am not entirely sure about the `language` entries but C/Python for a try.

### CODE REVIEW

**Comment by neteler** [comment]
`CodeQL did not detect any code written in languages supported by this CodeQL distribution`.

... ok, seems we need two minor code snippets in the repo to make  CodeQL happy.

**Comment by mmacata** [comment]
As there are two new issues, I removed the whole codeql.yml. In the [settings](https://github.com/mundialis/github-workflows/settings/security_analysis) it said:
"No CodeQL supported languages to scan in this repository"

Now it is activated again in the settings, even though no new file was created.

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      # only required for workflows in private repositories
+      actions: read
+      contents: read
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language:
+          - c-cpp
+          - python
```
Comment (neteler) [inline .github/workflows/codeql.yml:48]:
```suggestion
        language: [ 'c-cpp', 'python' ]
```

Comment (neteler) [reply .github/workflows/codeql.yml:48]: Inspired by
https://github.com/OSGeo/gdal/blob/5a7aac3e76f17c70f8eee1080ee120bb8fc0a6fc/.github/workflows/codeql.yml#L41

Comment (mmacata) [reply .github/workflows/codeql.yml]: Unfortunately this isn't the problem. It has something to do with compiling c-code I assume. I suggest to only check for python as we don't have this much c-code in the mundialis github organisation anyway!?

**Pull request merged**